### DISCUSSION TITLE
DatePicker Component May Be a Good Idea

### DISCUSSION THREAD

**Discussion by yehezkielgunawan** [tailwindlabs/headlessui#1105] [Ideas]
Just want to share an idea initiative here. Maybe it's good if there's also DatePicker Component in Headless UI that also can be customized based on the developer's preferences and need.

Reference: [Flowbite Datepicker](https://flowbite.com/docs/plugins/datepicker/#javascript)

**Comment by changesbyjames** [comment]
It's not a picker but I've had a lot of luck using [useLilius](https://github.com/its-danny/use-lilius) for making date-based components.

**Reply by yehezkielgunawan** [reply]
Actually, it's cool, but maybe it needs proper documentation and codes example (not only demo). Headless UI can make it so the devs who usually Tailwind don't need to install any additional libraries

**Comment by mohammadzolfali** [comment]
what should I do

در تاریخ چهارشنبه ۱۶ فوریهٔ ۲۰۲۲،‏ ۱۷:۴۶ yehezkielgunawan <
***@***.***> نوشت:
